README.md

```diff
@@ -152,6 +153,53 @@ reformatted connect/routes/api.py
 All done! ✨ 🍰 ✨
 1 file reformatted.
 ```
+## Build multi-arch docker image 
+LinuxForHealth connect runs on amd64, arm64 & s390x platforms.  Follow these instructions to build the multi-arch image that supports all these platforms.  At this time, you need access to all 3 platforms in order to build the multi-arch image.
+
+### Build the amd64 and arm64 image
+Run `docker buildx` to build an image that supports amd64 and arm64.  The s390x image has to be built separately, for now.
```

Comment:
Would you please take a look at your current docker version and bump our "minimum" recommended docker version to what you're using if necessary?